microservices/netinfra_utils/workers/common_worker.py

```diff
@@ -0,0 +1,68 @@
+import uniconfig_worker
+
+# Uniconfig RPCs
+
+
+def replace_cfg_with_oper(device_1, device_2):
+    return replace_cfg_with_oper2([device_1.id, device_2.id])
+
+
+def sync_from_net(device_1, device_2):
+    return sync_from_net2([device_1.id, device_2.id])
+
+
+def dryrun_commit(device_1, device_2):
+    return dryrun_commit2([device_1.id, device_2.id])
+
+
+def commit(device_1, device_2):
+    return commit2([device_1.id, device_2.id])
+
+
+def replace_cfg_with_oper2(device_ids):
```

Comment:
This is not a good practice. Provide just one method accepting an array and fix the callers